lib/charms/grafana_agent/v0/cos_agent.py

```diff
@@ -721,6 +721,8 @@ def metrics_jobs(self) -> List[Dict]:
                         "job_name": job["job_name"],
                         "metrics_path": job["path"],
                         "static_configs": [{"targets": [f"localhost:{job['port']}"]}],
+                        # We include insecure_skip_verify because we are always scraping localhost.
```

Comment:
Added more context in the comment. Makes sense?
```suggestion
                        # We include insecure_skip_verify because we are always scraping localhost.
                        # Even if we have the certs for the scrape targets, we'd rather specify the scrape
                        # jobs with localhost rather than the SAN DNS the cert was issued for.
```